src/Instrumentation/AwsSdk/src/AwsSdkInstrumentation.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+// src/Instrumentation/AwsSdk/AwsSdkInstrumentation.php
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\AwsSdk;
+
+use Aws\AwsClient;
+use Aws\Exception\AwsException;
+use Aws\ResultInterface;
+use OpenTelemetry\API\Instrumentation\CachedInstrumentation;
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Context\Context;
+use function OpenTelemetry\Instrumentation\hook;
+use OpenTelemetry\SemConv\TraceAttributes;
+
+final class AwsSdkInstrumentation
+{
+    public const NAME = 'aws-sdk';
+
+    public static function register(): void
+    {
+        $inst = new CachedInstrumentation(
+            'io.opentelemetry.contrib.php.aws-sdk',
+            null,
+            'https://opentelemetry.io/schemas/1.30.0',
+        );
+
+        /**
+         * ② Intercept the low‑level `execute` call that actually
+         *    performs the HTTP request and has the Command object.
+         */
+        hook(
+            AwsClient::class,
+            'execute',
+            pre: static function (
+                AwsClient $c,
+                array $params,
+                string $class,
+                string $function,
+                ?string $filename,
+                ?int $lineno
+            ) use ($inst) {
+                $cmd     = $params[0];
+                $builder = $inst->tracer()
+                    ->spanBuilder("{$c->getApi()->getServiceName()}.{$cmd->getName()}")
+                    ->setSpanKind(SpanKind::KIND_CLIENT)
+                    ->setAttribute(TraceAttributes::RPC_SYSTEM, 'aws-api')
+                    ->setAttribute(TraceAttributes::RPC_METHOD, $cmd->getName())
+                    ->setAttribute(TraceAttributes::RPC_SERVICE, $c->getApi()->getServiceName())
+                    ->setAttribute(TraceAttributes::CLOUD_REGION, $c->getRegion())
+                    ->setAttribute(TraceAttributes::CODE_FUNCTION_NAME, $function)
```

Comment:
Can you update this to use semver 1.32 ? function name is the FQN ($class::$function), CODE_NAMESPACE is removed, CODE_FILEPATH -> CODE_FILE_PATH, and the schema version up top.